CMakeLists.txt

```diff
@@ -72,6 +78,13 @@ set(spirv-compiler-options "")
 set(spirv-compiler-defines "")
 set(spirv-cross-link-flags "")
 
+if(SPIRV_CROSS_ENABLE_WEBMIN)
+    add_definitions(-DSPIRV_CROSS_WEBMIN)
```

Comment:
Changed ```add_definitions``` to follow the same pattern as ```spirv-compiler-defines```, the only difference is that we must introduce a ```spirv-public-compiler-defines```, because consuming targets must also apply the definition in order for the headers to look right.